formula1_race_analysis/custom_exceptions.py

```diff
@@ -0,0 +1,22 @@
+class CustomFileNotFoundError(Exception):
```

Comment:
I think the word Custom is redundant in these examples.

And I want you to build a hierarchy of Exceptions. When someone will use your package, and he wants to try except every possible error that your package might return (I mean those that you defined), he/she should be able to.
Define one base exception class that is consistent with naming of your package. And inherit the rest